spatialos.json

```diff
@@ -1,8 +1,8 @@
 {
     "name": "unity_gdk",
     "project_version": "0.0.1",
-    "sdk_version": "14.0-b6143-48ac8-WORKER-SNAPSHOT",
+    "sdk_version": "14.0-b6757-b205a-WORKER-SNAPSHOT",
     "dependencies": [
         {"name": "standard_library", "version": "14.0-b6143-48ac8-WORKER-SNAPSHOT"}
```

Comment:
Change this one for consistency - or remove it? Not sure what purpose this serves given we manually download the packages

Reply:
It's needed for a few things. Will discuss offline.

Reply:
Good point. The starter project needs to be changed too for the next release.
If we need to change the snapshot again we should just be able to take whichever is latest.

Reply:
but why is the standard library not changed?

Reply:
Oh that's a mistake